tests/jax/latest/common.libsonnet

```diff
@@ -85,4 +85,19 @@ local tpus = import 'templates/tpus.libsonnet';
       gsutil -m cp -r ${OUTPUT_DIR} $(MODEL_DIR) || exit 0
     ||| % (self.scriptConfig { extraFlags: std.join(' ', config.extraFlags) }),
   },
+
+  tpuEmbeddingCommon:: common.JaxTest {
+    local config = self,
+    frameworkPrefix: 'jax.tpu.embedding',
+    extraFlags:: [],
+    testCommand:: error 'Must define `testCommand`',
+    testScript:: |||
+      pip install --upgrade 'jax[tpu]==0.4.4' -f https://storage.googleapis.com/jax-releases/libtpu_releases.html
```

Comment:
This folder is for latest JAX version. Should we move tests to a folder named `4.4`, like TensorFlow tests? Or you could move it out of the latest folder, and along with other JAX unit tests: i.e. https://github.com/GoogleCloudPlatform/ml-testing-accelerators/blob/master/tests/jax/compilation-cache.libsonnet